src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$column-width: 100px;
+$gap: 1px;
+
+.calendar {
+  display: flex;
+  width: calc((7 * $column-width) + (6 * 1px));
```

Comment:
The width calculation here seems to be hardcoded for a 1px gap. Since you have a `$gap` variable, you should use it in the calculation instead of the hardcoded `1px` to maintain consistency and to make the code easier to maintain if the gap size changes in the future.